### DISCUSSION TITLE
Why not avoid translating when there is empty text instead of throwing an error?

### DISCUSSION THREAD

**Discussion by t4tianasc** [nidhaloff/deep-translator#131] [Q&A]
Hi, I've been using this library to translate a lot of small texts, so I got an error due to the limit of requests to Google, I've read the issues other people have raised but nothing has worked for me. Anyway, I'm now trying to use translate_batch, as suggested, and I would like to mention that I'm currently working with lots of records (so I leave a script running), but there are some of them which are empty, so then I got this exception:
```
line 48, in _validate_payload
    raise NotValidPayload(payload)
deep_translator.exceptions.NotValidPayload:  --> text must be a valid text with maximum 5000 character, otherwise it cannot be translated
```

Due to some reasons, in my particular case, it's a little bit difficult to treat the data to check before sending in `translate_batch(list)` the list without empty values as after I have to take the exact same list (with the empty ones) to assign in other place. Anyway, it would be just good I think if the library just returns an empty string when an empty string is sent to translate, I guess it's not hard to implement and it would make it much easier in my case, maybe some others, who knows.

Thanks for reading.

**Comment by nidhaloff** [comment]
Sorry for the late reply!
Yes, this was fixed in newer versions.
